web/templates/popovers/send_later_popover.hbs

```diff
@@ -1,62 +1,68 @@
-                    </span>
-                </div>
-            </label>
-            <label class="enter_sends_choice">
-                <input type="radio" name="enter_sends_choice" value="false"{{#unless enter_sends_true }} checked{{/unless}} />
-                <div class="enter_sends_choice_text">
-                    <span>
-                        {{#tr}}
-                            <z-shortcut></z-shortcut> to send
-                            {{#*inline "z-shortcut"}}<kbd>Ctrl</kbd>+<kbd>Enter</kbd>{{/inline}}
-                        {{/tr}}
-                    </span>
-                    <span class="enter_sends_minor">
-                        {{#tr}}
-                            <z-shortcut></z-shortcut> to add a new line
-                            {{#*inline "z-shortcut"}}<kbd>Enter</kbd>{{/inline}}
-                        {{/tr}}
-                    </span>
-                </div>
-            </label>
-        </div>
-    </li>
-    <hr />
-    <li>
-        <a class="open_send_later_modal" tabindex="0">{{t "Schedule message" }}</a>
-    </li>
-    {{#if formatted_send_later_time}}
-        <li>
-            <a class="send_later_selected_send_later_time" tabindex="0">{{t 'Schedule for {formatted_send_later_time}' }}</a>
+<div class="popover-menu" id="send_later_popover" data-simplebar data-simplebar-tab-index="-1">
+    <ul role="menu" class="popover-menu-list">
+        <li role="none" class="popover-menu-list-item">
+            <div role="group" class="enter_sends_choices" aria-label="{{t 'Enter to send choices' }}">
+                <label role="menuitemradio" class="enter_sends_choice" tabindex="0">
+                    <input type="radio" name="enter_sends_choice" value="true"{{#if enter_sends_true }} checked{{/if}} />
+                    <div class="enter_sends_choice_text_container">
+                        <span class="enter_sends_major enter_sends_choice_text">
+                            Press {{popover_hotkey_hints "Enter"}} to send
+                        </span>
+                        <span class="enter_sends_minor enter_sends_choice_text">
+                            Press {{popover_hotkey_hints "Ctrl" "Enter"}} to add a new line
+                        </span>
+                    </div>
+                </label>
+                <label role="menuitemradio" class="enter_sends_choice" tabindex="0">
+                    <input type="radio" name="enter_sends_choice" value="false"{{#unless enter_sends_true }} checked{{/unless}} />
+                    <div class="enter_sends_choice_text_container">
+                        <span class="enter_sends_major enter_sends_choice_text">
+                            Press {{popover_hotkey_hints "Ctrl" "Enter"}} to send
+                        </span>
+                        <span class="enter_sends_minor enter_sends_choice_text">
+                            Press {{popover_hotkey_hints "Enter"}} to add a new line
+                        </span>
```

Comment:
These strings seem to have lost their translation tags.

Reply:
Ah, my bad. Re-added them like such:
```handlebars
{{#tr}}
    Press <z-shortcut></z-shortcut> to send
    {{#*inline "z-shortcut"}}{{popover_hotkey_hints "Enter"}}{{/inline}}
{{/tr}}
```